### PR TITLE
add leads, execs to CODEOWNERS

### PR DESCRIPTION
**Pull request by bobheadxi**
# Pull Request

## Description

**Give a brief description of your changes:**

add leads, execs to CODEOWNERS - as a tool we want to invest in, Launch Pad leadership should be aware of the changes going through Rocket2.

Also renames the team to `@ubclaunchpad/rocket2`

## Testing

**If testing this change requires extra setup, please document it here:**
n/a

## Ticket(s)

n/a

(Create a copy of that line for each Github Issue affected,
and replace "Affects" with "Closes" if merging this will close the relevant ticket.)

### CODE REVIEW

**Comment by cheukyin699** [comment]
Looks like an issue with the codecov client version. Updating the client to the latest will solve this issue, as will #463 (which does the same thing, along with a few others).

**Comment by bobheadxi** [comment]
@cheukyin699 I'll wait until you merge that!

**Review by chuck-sys — APPROVED** [review]

**Pull request merged**
